velox/common/file/File.h

```diff
@@ -45,6 +41,8 @@
 
 namespace facebook::velox {
 
+constexpr std::string_view kFileScheme("file:");
```

Comment:
Curious why is this defined here and not where LocalFileSystem is defined?

Reply:
Will fix. Missed it while refactoring.